ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ip-security.md

```diff
@@ -0,0 +1,89 @@
+# ip-security
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+  - [IP Security](#ip-security)
```

Comment:
Should IP security be under management? I would have expected it's own main chapter. Or if we have a data plane security somewhere with macsec.